index.js

```diff
@@ -11,11 +13,12 @@ const awsCredentialsDeprecationNotice = function awsCredentialsDeprecationNotice
     'See: https://github.com/parse-server-modules/parse-server-s3-adapter#aws-credentials for details');
 };
 
-class S3Adapter {
+class S3Adapter extends FilesAdapter {
```

Comment:
This seemed like the right thing to do, even though the parse-server PR doesn't actually change the signature.